src/components/UI/Table/Table.vue

```diff
@@ -25,21 +25,41 @@
   <!-- structure inspired from https://tailwindui.com/components/application-ui/lists/tables  -->
   <div class="flex flex-col mx-10">
     <div class="-my-2 overflow-x-auto sm:-mx-6 lg:-mx-8">
-      <div class="py-2 align-middle inline-block min-w-full sm:px-6 lg:px-8">
-        <div class="shadow overflow-hidden border-b border-gray-200 sm:rounded-lg">
+      <div class="py-2 align-middle inline-block min-w-full px-4 sm:px-6 lg:px-8">
+        <div class="shadow overflow-hidden border-b border-gray-200 rounded-lg">
           <table class="min-w-full divide-y divide-gray-200">
             <thead class="bg-gray-200">
               <tr>
                 <th
-                  v-for="key in columns"
-                  @click="sortBy(key)"
-                  :key="key"
+                  v-for="(columnName, columnIndex) in columns"
+                  @click="sortBy(columnName)"
+                  :key="columnName"
+                  scope="col"
+                  class="sm:px-6 sm:py-3 px-3 py-1.5 text-left text-xs sm:text-md font-medium text-gray-500 uppercase tracking-wider"
+                  :class="{
+                    'hidden sm:table-cell': columnIndex != 0,
+                  }"
+                >
+                  <div class="flex">
+                    <div class="p-1 my-auto">
+                      {{ capitalize(columnName) }}
+                    </div>
+                    <div class="p-1 my-auto">
+                      <inline-svg
+                        :src="require('@/assets/images/chevron-down-solid.svg')"
+                        class="h-3 w-3 my-1"
+                      ></inline-svg>
+                    </div>
+                  </div>
+                </th>
```

Comment:
Will explain this component on call - yet to add comments and structure it in a better way